index.js

```diff
@@ -1,3 +1,4 @@
+const _ = require('lodash');
```

Comment:
I don't think we should use lodash here. it's not listed in package.json, which means every project that uses this lib has to have lodash, which is an unnecessary requirement in this case. Just do something like `(record) => record` instead of `_.identity`

Reply:
Yep, done